spec/team_result_parser_spec.rb

```diff
@@ -27,4 +29,117 @@
       expect(team_result_parser.percentage_ties).to eq (0.20)
     end
   end
+
+  describe "#offense" do
+    it "can find the best and worst offenses" do
+      league_parser = LeagueParser.new
+      team_result_parser = TeamResultParser.new
+
+      team_result_parser.get_game_info
+      league_parser.list_teams
+
+      expect(team_result_parser.best_offense(league_parser.teams_list)).to eq("Reign FC")
+      expect(team_result_parser.worst_offense(league_parser.teams_list)).to eq("Utah Royals FC")
```

Comment:
Not sure if its the view in here or do these two need to be tabbed over?

Reply:
Not sure I'm seeing that. I think it's good!